cmd/nodecmd/sync.go

```diff
@@ -51,17 +50,33 @@ func syncSubnet(_ *cobra.Command, args []string) error {
 	if _, err := subnetcmd.ValidateSubnetNameAndGetChains([]string{subnetName}); err != nil {
 		return err
 	}
-	isBootstrapped, err := checkNodeIsBootstrapped(clusterName, false)
+	notBootstrappedNodes, err := checkClusterIsBootstrapped(clusterName)
 	if err != nil {
 		return err
 	}
-	if !isBootstrapped {
-		return errors.New("node is not bootstrapped yet, please try again later")
+	if len(notBootstrappedNodes) > 0 {
+		return fmt.Errorf("node(s) %s are not bootstrapped yet, please try again later", notBootstrappedNodes)
 	}
-	if err := checkAvalancheGoVersionCompatible(clusterName, subnetName); err != nil {
+	incompatibleNodes, err := checkAvalancheGoVersionCompatible(clusterName, subnetName)
+	if err != nil {
 		return err
 	}
-	return trackSubnet(clusterName, subnetName, models.Fuji)
+	if len(incompatibleNodes) > 0 {
+		ux.Logger.PrintToUser("Either modify your Avalanche Go version or modify your Subnet-EVM version")
```

Comment:
Should this be Custom VM? Or are we guaranteed this can only happen with SubnetEVM?